server/src/internalClusterTest/java/org/opensearch/action/admin/cluster/stats/ClusterStatsIT.java

```diff
@@ -86,7 +86,8 @@ public void testNodeCounts() {
         internalCluster().startNode();
         Map<String, Integer> expectedCounts = new HashMap<>();
         expectedCounts.put(DiscoveryNodeRole.DATA_ROLE.roleName(), 1);
-        expectedCounts.put(DiscoveryNodeRole.MASTER_ROLE.roleName(), 1);
+        expectedCounts.put(DiscoveryNodeRole.MASTER_ROLE.roleName(), 1); // TODO: Remove along with MASTER_ROLE
```

Comment:
Nitpick, but TODO comments like this one just clutter the code in my opinion. When the `DiscoveryNodeRole.MASTER_ROLE` constant is removed the compiler won't by shy about telling what code needs to be cleaned up :) The deprecated annotation on that constant (and corresponding comment) are really all you need for this case.

Reply:
😄 Thanks for this opinion! I know I'm adding so many TODOs, I will listen to your suggestion. It make sense 😄, for this simple line of code, TODO comment are not necessary.

Reply:
I removed the unnecessary in the commit https://github.com/opensearch-project/OpenSearch/pull/2424/commits/e4404d7d27f82b0402378804ee7d5e644208e0c3